src/services/get-local-mods.ts

```diff
@@ -15,12 +16,26 @@ async function getLocalMods(): Promise<ModMap> {
       name: manifestFile.manifest.name,
       author: manifestFile.manifest.author,
       uniqueName: manifestFile.manifest.uniqueName,
-      folderName: manifestFile.path.replace(`${config.owmlPath}/Mods/`, '').split('/')[0],
+      modPath: path.dirname(manifestFile.path),
       localVersion: manifestFile.manifest.version,
       isLoading: false,
     },
   }), {});
 
+  const owmlManifestPath = `${config.owmlPath}/OWML.Manifest.json`;
+  const owmlManifest: Manifest = fs.existsSync(owmlManifestPath)
+    ? JSON.parse(fs.readFileSync(owmlManifestPath, { encoding: 'UTF-8' }))
+    : null;
+  const owml: Mod = {
+    name: owmlManifest?.name ?? 'OWML',
+    author: owmlManifest?.author ?? 'Alek',
+    uniqueName: owmlManifest?.uniqueName ?? 'Alek.OWML',
+    modPath: config.owmlPath,
+    localVersion: owmlManifest?.version ?? '< 0.3.43',
```

Comment:
If OWML version is below 0.3.43, the version number is not available to us. I check for "<" in isOutdated...